BytexDigital.Blazor.Components.CookieConsent/CookieConsentOptions.cs

```diff
@@ -197,7 +223,9 @@ public class CookieConsentCheckOptions
         {
             ["en"] = "This content uses cookies that are disallowed by your settings. To show it, we need to adjust your cookie settings for our website. We require your consent for:",
             ["de"] = "Dieser Inhalt nutzt Cookies, die aktuell nicht erlaubt sind. Um ihn anzuzeigen, müssen Sie Ihre Cookie-Präferenzen auf unserer Webseite aktualisieren. Wir benötigen Ihre Zustimmung für:",
-            ["nl"] = "Deze inhoud maakt gebruik van cookies die niet zijn toegestaan door uw instellingen. Om het te tonen, moeten we uw cookie-instellingen voor onze website aanpassen. Wij hebben uw toestemming nodig voor:"
+            ["nl"] = "Deze inhoud maakt gebruik van cookies die niet zijn toegestaan door uw instellingen. Om het te tonen, moeten we uw cookie-instellingen voor onze website aanpassen. Wij hebben uw toestemming nodig voor:",
+            ["es"] = "Este contenido utiliza cookies que no están permitidas por su configuración. Para mostrarlo, necesitamos ajustar la configuración de cookies para nuestro sitio web. Requerimos su consentimiento para:",
+            ["fr"] = "Ce contenu utilise des cookies qui sont interdits par vos paramètres.Pour l'afficher, nous devons ajuster vos paramètres de cookies pour notre site Web. Nous avons besoin de votre consentement pour :"
```

Comment:
Probably an unwanted space in "pour :"?

Reply:
In French there's a space preceding some punctuation marks (at least if you want to write according to the rules, since the rules are sometimes ignored) :)
https://mashable.com/article/leaving-space-before-end-punctuation-texting#:~:text=French.,Je%20d%C3%A9teste%20%C3%AAtre%20en%20ligne%20!

Reply:
Thanks! I'm not familiar with French grammar rules, so it seemed odd to my untrained eye.

Reply:
Sure np! I also find it weird sometime since it isn't my primary language. I do mostly English now, but yeah I try to adhere to these silly rules when working. I am glad I could help :) Great little tool btw.

Reply:
Glad you're finding the library useful 👍 Would you mind sharing how you found this repo? There seems to be more people finding it lately

Reply:
I just googled CookieConsent blazor to see what's out there. I needed it for a project that I am launching in the EU and I knew Cookie Consent was going to be an issue. Google then pointed me to the library and it seemed really like what I needed. It just needed French and Spanish for my use case so I decided to help out and add it.

Reply:
Thanks for the reply, nice to know it shows up in Google search results 👍 Your languages were published in version [1.0.13](https://www.nuget.org/packages/BytexDigital.Blazor.Components.CookieConsent/1.0.13)